gluster-utilization.libsonnet

```diff
@@ -0,0 +1,66 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'gluster-utilization',
+        rules: [
+          {
+            alert: 'GlusterVolumeUtilization',
+            expr: |||
+                  100 * gluster:volume_capacity_used_bytes_total:sum > 80,
```

Comment:
Isnt this metric `gluster:volume_capacity_used_bytes_total` no of bytes? If so how is percentage getting calculated here?